hw4/src/Coworker.java

```diff
@@ -0,0 +1,37 @@
+public class Coworker {
+    private String mName;
```

Comment:
Why you choose that naming?

Reply:
I used that name based on logic that mName is class member (prefix "m"). I saw such naming convention before. Which naming convention should we use?

Reply:
Changed naming according to our lessons.